modules/standard/Path.chpl

```diff
@@ -535,6 +549,7 @@ private proc normalizeLeadingSlashCount(name: string): int {
   :rtype: `string`
 */
 proc normPath(name: string): string {
+  use Lists;
```

Comment:
Since you add this use to more than three functions, I think it would be good to instead add a module-level `private use`